app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
         <% end %>
         <span class="caret"></span>
       </a>
-      <ul class="dropdown-menu" role="menu">
+      <ul class="dropdown-menu finders" role="menu">
```

Comment:
Thanks. Grabbing it with `'.dropdown-menu li'` also works but I guess `finders` does better describe its functionality.